docs/examples/howto/howto.rst

```diff
@@ -8,7 +8,7 @@ How-to guides
    run_material_mapping
    run_seeding
    run_sycl_seed_finding
-   run_truth_tracking_kalman
+   run_truth_tracking
```

Comment:
This is probably not fully related that the renaming comes here, but that's ok

Reply:
I can remove this.